en/releases/1.12.md

```diff
@@ -5,26 +5,65 @@
 
 ## Changes
 
+* [Common](#common)
+* [MAVLink](#mavlink)
+* [Commander](#commander)
 * [Multicoper](#multicopter)
+* [VTOL](#vtol)
+* [Control & Navigation](#control)
+
+### Common
+
+* **RTL Trigger based on remaining flight range ([PR#16399](https://github.com/PX4/PX4-Autopilot/pull/16399))**
+  * Calculates time to home, on RTL, taking into account vehicle speed, wind speed, and destination distance/direction
+* **Pre-emptive geofence breach ([PR#16400](https://github.com/PX4/PX4-Autopilot/pull/16400))**
+  * Triggers a breach if the _predicted_ current trajectory will result in a breach, allowing the vehicle to be re-routed to a safe hold position.
+
+### MAVLink
+
+* **MAVLink Ethernet configuration ([PR#14460](https://github.com/PX4/PX4-Autopilot/pull/14460))**
+  * MAVLink Ethernet channel settings such as UDP port, remote port and broadcast mode now can be changed dynamically via parameters.
+* **Support for querying `COMPONENT_INFORMATION` ([PR#16039](https://github.com/PX4/PX4-Autopilot/pull/16039))**
+  * This new message allows a GCS to request rich hierarchical information from an autopilot; i.e. in order to understand which commands are supported in missions, or to get parameter metadata (RFC: [mavlink#1339](https://github.com/mavlink/mavlink/issues/1339))
+
+### Commander
+
+* **Commander: use control mode flags and cleanup arm/disarm ([PR#16266](https://github.com/PX4/PX4-Autopilot/pull/16266))**
+  * Consolidates arming logic, and does a better job at keeping the current state
+* **Commander: Separate out manual control setpoint processing ([PR#16878](https://github.com/PX4/PX4-Autopilot/pull/16878))**
+  * Adds a new class `ManualControl` for handling `manual_control_setpoint` and handles RC loss, RC override, and RC arming/disarming
 
 ### Multicopter
 
-* #### Intuitive stick feel in Position mode
+* **More intuitive stick feel in Position mode**
   * Horizontal stick input mapped to acceleration instead of velocity setpoints
   * Removes unexpected tilt changes upon reaching travel speed velocity
   * Intuitive shunting e.g. when landing
-  * Comparable to Altitude mode but with the familiar benefits
-    * drift less hover
-    * straight fast forward flight
-    * automatic braking when letting go of the stick
   * Opt out possible using [MPC_POS_MODE](../advanced_config/parameter_reference.md#MPC_POS_MODE)
   * Development: [First attempt](https://github.com/PX4/PX4-Autopilot/pull/12072), [Introduction](https://github.com/PX4/PX4-Autopilot/pull/16052), [Improvements](https://github.com/PX4/PX4-Autopilot/pull/16320), [Bugfix zero oscillation](https://github.com/PX4/PX4-Autopilot/pull/16786), [Bugfix position unlock](https://github.com/PX4/PX4-Autopilot/pull/16791), [Bugfix invalid setpoint](https://github.com/PX4/PX4-Autopilot/pull/17078)
```

Comment:
What does all this mean? Are you saying that these parts of the same feature are still under development? Are they intended for this release?

Reply:
@MaEtUgR what's your intention behind the last list item? I'm wondering if there's a better way to represent this.